patches/net/minecraft/world/item/enchantment/EnchantmentHelper.java.patch

```diff
@@ -21,7 +21,7 @@
          return itemenchantments.getLevel(p_346179_);
      }
 @@ -122,6 +_,12 @@
-     private static void runIterationOnItem(ItemStack p_345425_, EnchantmentHelper.EnchantmentVisitor p_345023_) {
+     public static void runIterationOnItem(ItemStack p_345425_, EnchantmentHelper.EnchantmentVisitor p_345023_) {
```

Comment:
Am I missing something, why has this been patched to public but also seemingly has an AT to do the same thing

`public net.minecraft.world.item.enchantment.EnchantmentHelper runIterationOnItem(Lnet/minecraft/world/item/ItemStack;Lnet/minecraft/world/item/enchantment/EnchantmentHelper$EnchantmentVisitor;)V`

Feel like this patch should be removed if there's a matching AT

Reply:
This is not a patch, this is a change in the surrounding hunk from the result of the changed AT.  If it were a patch, there would be two change symbols (one of `++`, `--`, `+-` or `-+`)